scripts/fgi.sh

```diff
@@ -0,0 +1,269 @@
+#!/bin/bash
+# Copyright (c) The Diem Core Contributors
+# SPDX-License-Identifier: Apache-2.0
+set -e
+set -o pipefail
+
+TAG=""
+PR=""
+WORKSPACE=""
+ENV=""
+LOCAL_BUILD=""
+LOCAL_SWARM=""
+EXIT_CODE=0
+# Default timeout is 45 mins
+TIMEOUT_SECS=2700
+
+FORGE_K8S_CLUSTERS=(forge-1)
+K8S_CONTEXT_PATTERN='arn:aws:eks:us-west-2:853397791086:cluster/libra-CLUSTERNAME'
```

Comment:
Is the account ID in AWS public information?

Reply:
this info is hided by calling aws sts get-caller-identity instead